modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -182,7 +182,9 @@
 					_this.pagingInProgress = true;
 					_this.page++; // move to next page
 					mw.log("Playlist:: scrollEnd event. paging: true: trying to load next playlist page: page="+_this.page+", pageSize"+_this.getConfig('pageSize'));
-					var playlistRequest = {
+
```

Comment:
@amirch1 It's possible to resolve redundant call if remove L190-L192, but, I'll update file to use branching, like you've mentioned